tests/lib/eslint/eslint.js

```diff
@@ -3659,46 +3784,71 @@ describe("ESLint", () => {
         describe("don't ignore the entry directory.", () => {
             const root = getFixturePath("cli-engine/dont-ignore-entry-dir");
 
-            it("'lintFiles(\".\")' should not load config files from outside of \".\".", async () => {
-                ESLint = defineESLintWithInMemoryFileSystem({
-                    cwd: () => root,
+            let cleanup;
+
+            beforeEach(() => {
+                cleanup = () => { };
+            });
+
+            afterEach(async () => {
+                await cleanup();
+
+                const configFilePath = path.resolve(root, "../.eslintrc.json");
+
+                if (shell.test("-e", configFilePath)) {
+                    shell.rm(configFilePath);
+                }
+            });
+
+            it("'executeOnFiles(\".\")' should not load config files from outside of \".\".", async () => {
```

Comment:
Did you mean to change `lintFiles` to `executeOnFiles`?

```suggestion
            it("'lintFiles(\".\")' should not load config files from outside of \".\".", async () => {
```